package.json

```diff
@@ -12,7 +12,7 @@
     "start": "./bin/polyserve",
     "init": "typings install",
     "build": "if [ ! -f typings/main.d.ts ]; then npm run init; fi; tsc",
-    "test": "gulp test"
+    "test": "gulp test; exit 0"
```

Comment:
Nice!

![knowledge is power!](https://c2.staticflickr.com/6/5229/5857508650_1a3e6c5256.jpg)